frame/transaction-payment/src/lib.rs

```diff
@@ -656,6 +656,19 @@ where
 	}
 }
 
+impl<T: Config, AnyCall: GetDispatchInfo + Encode> EstimateCallFee<AnyCall, BalanceOf<T>>
```

Comment:
Should this be called `MaxCallFee`?

Estimate implies to me it could be more or less, but really it can only be less than this fee.

Reply:
I'm trying to take into account that other implementations might differ, and might be more or less. The implementation by `transaction-payment` is actually accurate to the point (excluding tip). 

But the trait name should not be moulded by the implementation detail, IMO in any case.